### ISSUE TITLE
Search box suggestions

### ISSUE THREAD

**Issue by gbrn** [WWBN/AVideo#8954]
@DanielnetoDotCom Danny, I think of a more interesting use for the search box.
See this interesting example... when clicking on the search box, the user already receives some suggestions based on their uses.

![image](https://github.com/WWBN/AVideo/assets/37880073/153c7370-b1df-48b4-b666-66edf2b76be9)

**Comment by DanielnetoDotCom** [comment]
Good idea, but for now we do not save the search history. but I may add it in the future

**Comment by gbrn** [comment]
@DanielnetoDotCom Danny, this search suggestion is more related to what the user consumes, rather than what the user searches for. This list of suggestions is independent of what the user is searching for.

In this interesting video, it shows how NetfIix has used, practical suggestions for users:
[2024-05-07 At 22.08.29.webm](https://github.com/WWBN/AVideo/assets/37880073/3b6a2698-e0de-471a-b0fc-9ad91d66106f)
